File: Q1.py
import time

def main():
    
    nums = []

    for x in range(1000):
        
        if is_multiple(x, 3) == True:
            nums.append(x)
        elif is_multiple(x, 5) == True:
            nums.append(x)
        else:
            pass
    
    print(f"Total sum: {sum(nums)}")


def is_multiple(x: int, a: int):

   if x % a == 0:
       return True
   else:
       return False

start = time.time()
main()
end = time.time()
print(start-end)

File: Q1.jl
module Q1

export main

function main()
    
    nums = []

    for x in 0:999
        if is_multiple(x, 3) == true
            push!(nums, x)
        elseif is_multiple(x, 5) == true
            push!(nums, x)
        end
    end

    println("Total sum: $(sum(nums))")

end

function is_multiple(x:: Int, a:: Int)

    return x % a == 0

end

end
